tests/test_misc_table.py

```diff
@@ -219,6 +225,13 @@ def test_dtype_column(
             [],
             str,
             'object',
+            audformat.define.DataType.OBJECT,
```

Comment:
This is a little bit unfortunate, but as long as `pandas` decide that the following is correct we cannot change it:

```python
>>> pd.Series([], dtype=str)
Series([], dtype: object)
```

and need to force the users to use

```python
>>> pd.Series([], dtype='string')
Series([], dtype: string)
```

instead.

Reply:
Yes, I also don't see another solution for it. We should probably add to `audformat.MiscTable.__init__()` that a user should explicitly use `dtype='string'` if a level contains string values as otherwise the type will be set to `object`.